### DISCUSSION TITLE
v0 Refund

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#7334] [Help] [answered]
### Summary

Can I get a refund of v0 Premium?
I made that purchase by mistake and I have NOT used v0 yet.

### Steps to Reproduce

Purchased v0 by mistake

**Comment by pawlean** [comment, marked as answer]
Hi @Junhojang13!

Are you a Pro customer? If so, can [submit a support case](https://vercel.com/help)?

**Reply by ghost** [reply]
I am using Pro, and I submitted case. Thank you for helping me.

**Reply by pawlean** [reply]
No worries! Here if you have any more questions.

[a comment by an automated account is not shown]
